src/System.Management.Automation/help/HelpUtils.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Help;
+using Microsoft.PowerShell.Commands;
+
+namespace System.Management.Automation
+{
+    internal class HelpUtils
+    {
+        private static string userHomeHelpPath = null;
+
+
+        /// <summary>
+        /// Get the path to $HOME
+        /// </summary>
+        internal static string GetUserHomeHelpSearchPath()
+        {
+            if (userHomeHelpPath == null)
+            {
+#if UNIX
+                var userModuleFolder = Platform.SelectProductNameForDirectory(Platform.XDG_Type.USER_MODULES);
+                string userScopeRootPath = System.IO.Path.GetDirectoryName(userModuleFolder);
+#else
+                string userScopeRootPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "PowerShell");
+#endif
+                userHomeHelpPath = Path.Combine(userScopeRootPath, "Help");
```

Comment:
Do we need to worry about SxS PSCore6.x?

Reply:
Fixed. Added a PSVersion string in path.

Reply:
Reverted this change, since the source location for help can only host 1 help content version. So even if we have SxS PowerShell versions, the content only has 1 published version.